config/default.yml

```diff
@@ -511,6 +511,7 @@ Style/TrailingComma:
   EnforcedStyleForMultiline: no_comma
   SupportedStyles:
     - comma
+    - strict_comma
```

Comment:
This will have to be explained in the preceding comment. I'm also not sure about this name.